plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -1814,6 +1829,35 @@ GraphHandlePtr QtOSGViewer::drawtrimesh(const float* ppoints, int stride, const
     return GraphHandlePtr(new PrivateGraphHandle(shared_viewer(), handle));
 }
 
+class GlobalLOD : public osg::LOD
```

Comment:
You have two declarations / implementations of GlobalLOD, one in qtosgviewer and another in osgrenderitem.cpp/osgrenderitem.h. Dont reimplement same class twice. What I meant was to create a separate .h / cpp file for our new OSGLODLabel. Then you just use #include "OSGLODLabel.h". Also, add OSGLODLabel.h and OSGLODLabel.cpp in plugins/qtosgrave/CMakeLists.txt (line 24).

We want only one implementation.

Also, why did you create a nested class for GlobalLOD? OSGLODLabel can directly inherit from LOD, dont need extra nested class, do we?